etc/docker/hangar-test-maven/include/project-1/test-1.bats

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bats
+
+@test "Compiled target jar exists" {
+  result="$(ls -l ./project-1/target/hangar-test-0.0.1-SNAPSHOT.jar | wc -l)"
+  [ "$result" -eq 1 ]
```

Comment:
Yep - that was a line that was reading all of the artifacts I was downloading, and then I decided it was rubbish because the transitive dependencies might change so I changed it, but not enough of a change. 

Will fix tomorrow morning...